mkt/reviewers/views.py

```diff
@@ -1005,6 +1006,27 @@ def post(self, request, pk, region, *args, **kwargs):
         return Response({'approved': bool(form.cleaned_data['approve'])})
 
 
+class _WebsiteAction(object):
+    permission_classes = [GroupPermission('Apps', 'Review')]
```

Comment:
Eventually we will need a `Website Reviewers` group eventually, with this permission added to it.  Also add `Websites:*` to `Staff` group.